Utils/gitlab_triggers/trigger_test_upload_flow.sh

```diff
@@ -97,57 +92,27 @@ if [[ -n "$_storage_base_path" ]] && [ "$_storage_base_path" != upload-flow* ];
   exit 1
 fi
 
-if [ -n "$_gitlab" ]; then
-
-  _variables="variables[BUCKET_UPLOAD]=true"
-  if [ -n "$_force" ]; then
-    _variables="variables[FORCE_BUCKET_UPLOAD]=true"
-  fi
-
-  if [ -z "$_override_all_packs" ]; then
-    _override_all_packs=false
-  else
-    _override_all_packs=true
-  fi
-
-  source Utils/gitlab_triggers/trigger_build_url.sh
 
-  curl --request POST \
-    --form token="${_ci_token}" \
-    --form ref="${_branch}" \
-    --form "${_variables}" \
-    --form "variables[SLACK_CHANNEL]=${_slack_channel}" \
-    --form "variables[PACKS_TO_UPLOAD]=${_packs}" \
-    --form "variables[GCS_MARKET_BUCKET]=${_bucket}" \
-    --form "variables[IFRA_ENV_TYPE]=Bucket-Upload" \
-    --form "variables[STORAGE_BASE_PATH]=${_storage_base_path}" \
-    --form "variables[OVERRIDE_ALL_PACKS]=${_override_all_packs}" \
-    "$BUILD_TRIGGER_URL"
+_variables="variables[BUCKET_UPLOAD]=true"
+if [ -n "$_force" ]; then
+  _variables="variables[FORCE_BUCKET_UPLOAD]=true"
+fi
 
+if [ -z "$_override_all_packs" ]; then
+  _override_all_packs=false
 else
-
-  trigger_build_url="https://circleci.com/api/v2/project/github/demisto/content/pipeline"
-
-  post_data=$(cat <<-EOF
-  {
-    "branch": "${_branch}",
-    "parameters": {
-      "gcs_market_bucket": "${_bucket}",
-      "bucket_upload": "${_bucket_upload}",
-      "force_pack_upload": "${_force}",
-      "packs_to_upload": "${_packs}",
-      "slack_channel": "${_slack_channel}",
-      "storage_base_path": "${_storage_base_path}"
-    }
-  }
-  EOF
-  )
-
-  curl \
-  --header "Accept: application/json" \
-  --header "Content-Type: application/json" \
-  -k \
-  --data "${post_data}" \
-  --request POST ${trigger_build_url} \
-  --user "$_ci_token:"
+  _override_all_packs=true
 fi
+
+source Utils/gitlab_triggers/trigger_build_url.sh
+
+curl --request POST \
+  --form token="${_ci_token}" \
+  --form ref="${_branch}" \
+  --form "${_variables}" \
+  --form "variables[SLACK_CHANNEL]=${_slack_channel}" \
+  --form "variables[PACKS_TO_UPLOAD]=${_packs}" \
+  --form "variables[GCS_MARKET_BUCKET]=${_bucket}" \
+  --form "variables[IFRA_ENV_TYPE]=Bucket-Upload" \
+  --form "variables[STORAGE_BASE_PATH]=${_storage_base_path}" \
```

Comment:
what happened to 
```    --form "variables[OVERRIDE_ALL_PACKS]=${_override_all_packs}" \```?

Reply:
It was a result of a bad merge.
Good catch.